docs/src/content/docs/de/components/tabs.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Registerkarten
+description: Lerne, wie du in Starlight Oberflächen mit Registerkarten erstellst, um gleichwertige Informationen zu gruppieren.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Um eine Oberfläche mit Registerkarten zu erstellen, verwende die Komponenten `<Tabs>` und `<TabItem>`.
+Registerkarten sind nützlich, um gleichwertige Informationen zu gruppieren, wenn ein Nutzer nur eine von mehreren Optionen sehen muss.
+
+import Preview from '~/components/component-preview.astro';
+
+<Preview>
+
+<Tabs slot="preview">
+	<TabItem label="Stars">Sirius, Wega, Betelgeuse</TabItem>
+	<TabItem label="Moons">Io, Europa, Ganymed</TabItem>
```

Comment:
I think you translated these labels in a few other pages so I guess we could do the same here (multiple occurrences in this page).